_quarto.yml

```diff
@@ -12,7 +12,6 @@ project:
     - outputs
     - display-messages
     - docs
-    - in-depth
```

Comment:
Removed `in-depth/` folder since the content seemed redundant and not used/finished